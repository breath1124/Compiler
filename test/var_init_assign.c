int main()
{
    int a = 1;
    int b = 2.2;
    int c = 'c';
    print a;
    print b;
    print c;
}