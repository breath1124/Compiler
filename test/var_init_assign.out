24 19 0 5 25 13 0 1 12 13 0 1 1 26 1074580685 12 13 0 2 1 27 99 12 13 11 22 15 -1 13 0 1 1 11 22 15 -1 13 0 2 1 11 22 21 3